WalletWasabi.Gui/Controls/TogglePasswordBox.cs

```diff
@@ -0,0 +1,61 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Styling;
+using System;
+using ReactiveUI;
+using System.Reactive.Linq;
+
+namespace WalletWasabi.Gui.Controls
+{
+	public class TogglePasswordBox : ExtendedTextBox, IStyleable
+	{
+		Type IStyleable.StyleKey => typeof(TogglePasswordBox);
+
+		public static readonly StyledProperty<bool> IsPasswordVisibleProperty =
+			AvaloniaProperty.Register<TogglePasswordBox, bool>(nameof(IsPasswordVisible), defaultBindingMode: BindingMode.TwoWay);
```

Comment:
I think the `defaultBindingMode` is already `TwoWay`? so this might be not necessary. Please do check though.

Reply:
`BindingMode defaultBindingMode = BindingMode.OneWay`

Reply:
Alright, good to know :)